### DISCUSSION TITLE
Shutdown dialog on Windows 11

### DISCUSSION THREAD

**Discussion by splisp** [Open-Shell/Open-Shell-Menu#1883] [General]
I am installing OpenShell on a new machine and it's Windows 11. I cannot see the "Shutdown dialog" item in the start menu... I have even tried adding a Custom item with the shutdown or shutdown_box command but there's no way. An item doesn't appear in the menu. Anybody else facing this issue?

**Comment by ge0rdi** [comment]
What _menu style_ are you using?

In `Classic` and `Classic with two columns` styles there is `Shutdown` button by default that shows shutdown dialog.
In `Windows 7` style you can select what _shutdown_ action the button does (instead of showing shutdown dialog).

Can you post screenshot of your menu for clarification?

**Reply by splisp** [reply]
Unfortunately (or, luckily!) I am out of office until Wednesday. I'll post screenshots and more details when I'm back, thanks!

**Reply by splisp** [reply]
Here are the screenshots from my PC:

![ClassicShell-Menu](https://github.com/Open-Shell/Open-Shell-Menu/assets/2437263/76fe525f-2e2f-4839-ac07-1ec8bc4708f5)
![ClassicShell-Setttings-1](https://github.com/Open-Shell/Open-Shell-Menu/assets/2437263/66dfa7f6-c057-4481-ba57-d0e634cfed4c)
![ClassicShell-Setttings-2](https://github.com/Open-Shell/Open-Shell-Menu/assets/2437263/62333da1-a4a4-47bb-b414-b610a0576854)

**Reply by splisp** [reply]
I didn't re-add the items under the "Shutdown dialog" when I took the screenshot... anyway, I did it and I also added the Search Box but still I can't get the Shutdown items in my Start menu:

![ClassicShell-Setttings-2b](https://github.com/Open-Shell/Open-Shell-Menu/assets/2437263/33df3231-d8e2-486d-a25f-7df013571514)

**Reply by splisp** [reply]
I have the very same configuration on my Windows 10 machine and I do see the Shutdown items there. I had actually started importing a backup exported from there.

**Comment by ge0rdi** [comment]
Apparently you are connected via RDP (remote desktop).
In such case `Open-Shell` doesn't display Shutdown box by default.

You can enable it by enabling `Show Shutdown during remote sessions` setting.

**Reply by splisp** [reply]
Yes, I am on RDP... How did you spot it? Great that you did! Thanks!
